sentinel/server.js

```diff
@@ -27,7 +27,7 @@ const waitForApi = () =>
         `http://localhost:${process.env.API_PORT || 5988}/setup/poll`,
         (err, response, body) => {
           if (err) {
-            logger.info('Waiting for API to be ready...');
+            logger.info('Waiting for API to be ready…');
```

Comment:
Using "..." is preferable to ellipsis because it's ASCII.

Reply:
This totally makes sense! I was mirroring existing style on the api [server.js](https://github.com/medic/cht-core/blob/master/api/server.js#L33).  Should I just revert this or revert this _and_ fix other instances of it (mainly in `api/server.js`)? 

Fun fact, [you can't search for `…` in github](https://github.com/medic/cht-core/search?q=%E2%80%A6&unscoped_q=%E2%80%A6) ;)

Reply:
It's perfectly fine to just revert this and not "fix" the others. I should update the style code some time…

Reply:
I'm requesting a re-review b/c I think that's the right workflow? It's only for this 1 char -> 3 char change I made, so lemme know if that's not right.